Commands/ShowCount.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Discord;
+using Discord.WebSocket;
+using FFXIVVenues.Veni.Commands.Brokerage;
+using FFXIVVenues.Veni.Context;
+using FFXIVVenues.Veni.Intents;
+using System.Threading.Tasks;
+
+namespace FFXIVVenues.Veni.Commands
+{
+    internal class ShowCount
+    {
+        public const string COMMAND_NAME = "showcount";
+
+        internal class CommandFactory : ICommandFactory
+        {
+            public SlashCommandProperties GetSlashCommand (SocketGuild guildContext = null)
+            {
+                return new SlashCommandBuilder()
+                    .WithName(COMMAND_NAME)
+                    .WithDescription("Show how many venues we have indexed so far")
```

Comment:
Generally don't use `we` since other people that aren't staff can run this too. Also, full stop!